src/main/java/Duke.java

```diff
@@ -1,10 +1,387 @@
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.lang.Exception;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.charset.StandardCharsets;
+import java.time.LocalDateTime;
+
+class DukeException extends Exception {
+    public DukeException(String s) {
+        super(s);
+    }
+}
+
 public class Duke {
+    private ArrayList<Task> todoList;
+    File saveList;
+    FileWriter editor;
+
+    public Duke() {
+        this.todoList = new ArrayList<Task>();
+    }
+
+    private void initialize(){
+        saveList=new File("src/main/data/list.txt");
+        boolean isNew;
+        try {
+            isNew = saveList.createNewFile();
+            if(!isNew){
+                Scanner sc = new Scanner(saveList);
+                this.transferSavedFile(sc);
+                editor = new FileWriter(saveList, true);
+                print("Save file loaded");
+            }
+        }catch(IOException e){
+            print("Directory search error. No save file loaded. Initialized with blank list. No commands will be saved to hard disk");
+
+        }
+    }
+
+    private void transferSavedFile(Scanner sc){
+        int index = 0;
+        while(sc.hasNext()){
+            String saveEntry = sc.nextLine();
+            String instruction[] = saveEntry.split(" \\| ");
+            if(instruction[0].equals("T")){
+                Todo task = new Todo(instruction[2]);
+                todoList.add(task);
+            }else if(instruction[0].equals("E")){
+                Event task = new Event(instruction[2], instruction[3]);
+                todoList.add(task);
+            }else if(instruction[0].equals("D")){
+                Deadline task = new Deadline(instruction[2], instruction[3]);
+                todoList.add(task);
+            }
+            if(instruction[1].equals("1")){
+                todoList.get(index).silentSetDone();
+            }
+            index++;
+        }
+        sc.close();
+    }
+
+    private static void print(String message) {
+        System.out.println(">>" + message);
+    }
+
+    private void list() {
+        StringBuilder outputMessageMessage = new StringBuilder("List:");
```

Comment:
outputMessageMessage does not communicate the intent of the StringBuilder object.
consider using a clearer name like outputMessageBuilder.